core/src/test/java/org/jclouds/rest/internal/BaseRestApiExpectTest.java

```diff
@@ -92,22 +92,22 @@
 import com.google.inject.TypeLiteral;
 
 /**
- * 
+ *
```

Comment:
Can you remove these spurious changes?

Reply:
I think so, but I'll wait until I get a response on the exception handling point before making those changes.

Reply:
As I wrote above, instead I'm going to commit now so you have something concrete to respond to.

Reply:
Done